src/components/editor/readonlyQuillContent.ts

```
@@ -0,0 +1,959 @@
+
+    const paragraphs = Array.from(root.querySelectorAll('p'))
+    for (const paragraph of paragraphs) {
+      const paragraphText = (paragraph.textContent ?? '')
+        .replace(/\u00a0/g, ' ')
+        .trim()
+      let candidateUrl: string | null = null
+
+      const links = Array.from(paragraph.querySelectorAll('a[href]'))
+      if (links.length === 1) {
+        const clone = paragraph.cloneNode(true) as HTMLParagraphElement
+        clone.querySelectorAll('a[href]').forEach(linkNode => linkNode.remove())
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)